test/fixture/webpack/src/app.jsx

```diff
@@ -19,10 +19,13 @@ function Test() {
   )
 }
 
+export const Anchor = 'a';
+
 export function App(props) {
   return (
     <Style id="color">
       <Test />
+      <Anchor />
```

Comment:
FWIW, locally this example does not fail for me when I try something similar in a modified version of the webpack example. And this example also does not produce any failing tests before the fix.